src/server/templates/model.html

```diff
@@ -52,132 +52,129 @@
 </style>
 {% endblock %} {% block content %}
 
+<div class="container-wrapper">
```

Comment:
FYI: This class/structure change breaks the column scrolling PR I just merged. (Admittedly, that PR is brittle, but I didn't want to do a refactor of this magnitude while I wasn't yet familiar with the code and team practices.)

Please update [those styles](https://github.com/epidemics/covid/blob/eb50a7a6efc90bd3df55bf113d97d10b8a61edfc/src/server/templates/model.html#L53-L69) so scrolling continues to work. If you choose to eliminate the bootstrap grid, you can also remove the media query.

Reply:
Resolved.

Reply:
Resolved